operate/webapp/src/main/java/io/camunda/operate/webapp/security/identity/IdentityPermission.java

```diff
@@ -9,7 +9,12 @@
 
 public enum IdentityPermission {
   READ,
+  READ_DECISION_INSTANCE,
+  READ_PROCESS_DEFINITION,
+  READ_PROCESS_INSTANCE,
   DELETE,
   UPDATE_PROCESS_INSTANCE,
-  DELETE_PROCESS_INSTANCE;
+  DELETE_PROCESS_INSTANCE,
+  DELETE_DECISION_INSTANCE,
+  DELETE_PROCESS;
```

Comment:
:wrench: I feel like this should be `DELETE_PROCESS_DEFINITION` to stay consistent with `READ_PROCESS_DEFINITION`

Reply:
I really am not ready to start a discussion about the permissions again. I followed the document 🤷

Reply:
:laughing: Alright then.